backend/src/services/encrypt/encrypt.service.ts

```diff
@@ -0,0 +1,14 @@
+import { genSalt, hash } from 'bcrypt';
+import { USER_PASSWORD_SALT_ROUNDS } from '~/common/constants/constants';
+
+class Encrypt {
+  generateSalt(): Promise<string> {
+    return genSalt(USER_PASSWORD_SALT_ROUNDS);
```

Comment:
```suggestion
    return genSalt(this.#salt);
```
pass salt as constructor's param